samples/dashboards/cypress/options-lang/demo.html

```diff
@@ -0,0 +1,7 @@
+<!-- <script src="https://code.highcharts.com/dashboards.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
```

Comment:
Remove?

Reply:
Done, but it was not a part of this task ;)